processes/releases.md

```diff
@@ -0,0 +1,42 @@
+# New MeiliSearch releases - what to do
+
+Those are the steps we need to take every time we release a new version of MeiliSearch.
+
+## Responsibilities
+
+Right now, unless mentioned, the tasks for a new release on the developer relations side are divided in three categories or actions with each person accountable for them:
```

Comment:
```suggestion
Right now, unless mentioned otherwise, the tasks for a new release on the developer relations side are divided into three categories or actions with each person accountable for them:
```